R/MsExperiment-functions.R

```diff
@@ -305,3 +305,19 @@ sampleData  <- function(object) {
     } else x <- x[i]
     x
 }
+
+#' @export
+#'
+#' @rdname MsExperiment
+qdata <- function(object) {
+    object@assay
```

Comment:
What about testing the class with `stopifnot(inherits(object, "MsExperiment"))` like in the replacement function?

Reply:
Now fixed in the last commit